app/models/category.php

```diff
@@ -0,0 +1,25 @@
+<?php
+class Category extends AppModel
+{
+    const TABLE_NAME = 'category';
+
+    public static function getAll()
+    {
+        $db = DB::conn();
+        $rows = $db->rows(sprintf('SELECT * FROM %s', self::TABLE_NAME));
```

Comment:
I agree with fatima, if we are to consider micro - optimization , functions (in this case sprintf()) uses more memory compared to string. Also declaring a class constant (self::TABLE_NAME) consume memory. I think its better to use string :smile: